src/metrics/api_v2_views.py

```diff
@@ -32,7 +33,7 @@ def fn(v):
 
     def isin(lst):
         def fn(val):
-            ensure(val in lst, "value %r is not in %r" % (val, lst))
```

Comment:
fixes a vulnerability that was also present in lax but fixed after security check